homeassistant/components/onewire/quality_scale.yaml

```diff
@@ -0,0 +1,126 @@
+rules:
+  ## Bronze
+  config-flow:
+    status: todo
+    comment: Under review
```

Comment:
What is?

Reply:
I've reviewed - and I think it's ready now.

Reply:
> I've reviewed - and I think it's ready now.

Actually it's not - it's still missing data descriptions in options flow
Adjusted status back to todo with corresponding comments